src/Service/ContainerAbstractServiceFactory.php

```diff
@@ -9,8 +9,8 @@
 
 namespace Zend\Session\Service;
 
-use Zend\ServiceManager\AbstractFactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use Interop\Container\ContainerInterface;
```

Comment:
container-interop should be added as a dev dependency. Yes, the SM depends on it as well, but you're explicitly referencing it in code here, which makes it another dependency.